drivers/wifi/simplelink/simplelink.c

```diff
@@ -212,6 +212,8 @@ static struct net_offload simplelink_offload = {
 	.put	       = NULL,
 };
 
+int simplelink_socket_create(int family, int type, int proto);
+
```

Comment:
It will be much safer to declare that function in `simplelink_support.h` file, so that compiler/linker will detect if there is difference between declaration and definition.

Reply:
Indeed, I've moved both declarations to respective headers.